packages/itwinui-react/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 3.5.0
+
+### Minor Changes
+
+- [#1844](https://github.com/iTwin/iTwinUI/pull/1844): Interactive elements will now be correctly focused on Safari.
```

Comment:
One leftover comment from #1844 that can be addressed here:

> Maybe the version can be mentioned here.
> > Oop, your comment came in at the same time as I hit "merge" 😅 We can update the changelog before making the final release.
> > 
> > Can you leave a comment on #1853?

_Originally posted by @mayank99 in https://github.com/iTwin/iTwinUI/pull/1844#discussion_r1488688041_

Reply:
```suggestion
- [#1844](https://github.com/iTwin/iTwinUI/pull/1844): Interactive elements will now be correctly focused on Safari 17+.
```